diff --git a/configure b/configure
index 65e7d48..c85e89d 100755
--- a/configure
+++ b/configure
@@ -15335,7 +15335,7 @@ $as_echo_n "checking for boostlib >= $boost_lib_version_req... " >&6; }

                         libsubdirs="lib"
     ax_arch=`uname -m`
-    if test $ax_arch = x86_64 -o $ax_arch = ppc64 -o $ax_arch = s390x -o $ax_arch = sparc64; then
+    if test $ax_arch = x86_64 -o $ax_arch = aarch64 -o $ax_arch = ppc64 -o $ax_arch = s390x -o $ax_arch = sparc64; then
         libsubdirs="lib64 lib lib64"
     fi

